client/my-sites/sidebar/index.jsx

```diff
@@ -769,25 +724,6 @@ export class MySitesSidebar extends Component {
 		);
 	}
 
-	trackMenuItemClick = ( menuItemName ) => {
-		this.props.recordTracksEvent(
-			'calypso_mysites_sidebar_' + menuItemName.replace( /-/g, '_' ) + '_clicked'
-		);
-	};
-
-	trackPlanClick = () => {
-		this.trackMenuItemClick( 'plan' );
-		this.props.recordTracksEvent( 'calypso_upgrade_nudge_cta_click', {
-			cta_name: 'sidebar_upgrade_default',
-		} );
```

Comment:
This is where we are triggering the nudge cta click event. This function is only called by the domains menu item.